saos-webapp/src/main/webapp/WEB-INF/view/template/pageDescription.jsp

```diff
@@ -1,13 +0,0 @@
-<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
-
-<tiles:importAttribute name="metaPageDescriptionCode" />
-
-<%-- Page description only for the root of the given page, for example for /search but not for /search?dateFrom... 
-     Search engines usually omit page descriptions if they are the same for many pages 
---%>
-<c:if test="${empty pageContext.request.parameterMap}"> 
-	<c:if test="${!empty metaPageDescriptionCode}">
-	        <meta name="description" content="<spring:message code='${metaPageDescriptionCode}'/>"/>
-	</c:if>
-</c:if>
```

Comment:
This code above is important. The meta description cannot be the same for different pages. Now we have the same description for /search and /search?judgeName="Sylwester+Kuchnio". Google won't allow this because it would look strange to have many result links with the same description.